distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -33,6 +33,7 @@
         <exclude>etc/org.apache.karaf.log.cfg</exclude>
         <exclude>etc/org.ops4j.pax.web.cfg</exclude>
         <exclude>etc/users.properties</exclude>
+        <exclude>etc/startup.properties</exclude>
```

Comment:
Updating the file on the fly would be considerably more effort than it's worth for this. We will hopefully be able to get rid of our custom `startup.properties` soon, as Karaf is going to move to log4j2 as its default in an upcoming release.